dbms/src/Flash/FlashService.h

```diff
@@ -89,6 +89,13 @@ class FlashService : public tikvpb::Tikv::Service
     grpc::Status tryAddLock(grpc::ServerContext * /*context*/, const disaggregated::TryAddLockRequest * request, disaggregated::TryAddLockResponse * response) override;
     grpc::Status tryMarkDelete(grpc::ServerContext * /*context*/, const disaggregated::TryMarkDeleteRequest * request, disaggregated::TryMarkDeleteResponse * response) override;
 
+    // Build the disaggregated task on the TiFlash write node,
```

Comment:
Seems that the pairing is:

Read Node / Write Node
Compute Node / Storage Node